### ISSUE TITLE
sharing weights of the pretrained stage model

### ISSUE THREAD

**Issue by jasonppy** [NVIDIA/audio-flamingo#13]
Hi Zhifeng,

Since there are a few datasets that I cannot obtain, is it possible to share the weights of the model after the pretraining stage? 

Also this is the training loss of pretraining stage (quite high variance):
<img width="380" alt="image" src="https://github.com/user-attachments/assets/b70a64fe-cb34-4c92-b309-87f344f208a7">

and this is the training loss of the SFT stage
<img width="344" alt="image" src="https://github.com/user-attachments/assets/bfa5e8a1-ed72-4af3-8ee2-c9e4b3bc8b43">

Do them look right to you?

My reproduced results are a bit far from reported results. And sharing the weights of the model after the pretraining stage will great help me narrow down the issue.

Appreciate your time and effort!

Puyuan

**Comment by zhifengkongnv** [comment]
We could not opensource the additional model checkpoint due to review policy. 

The training loss (SFT) looks fine to me - if you set smoothing=0.6 then it oscillates between 0.5 and 1.5. 

<img width="1059" alt="Screenshot 2024-08-13 at 2 46 31 PM" src="https://github.com/user-attachments/assets/af3ea5fe-a6ff-4a5e-9aad-8231e0c4287a">

The validation loss is more informative. For example, the valid_losses of 3 different validation sets below indicate the model learns well on these tasks.

<img width="1054" alt="Screenshot 2024-08-13 at 2 50 14 PM" src="https://github.com/user-attachments/assets/96fcd62e-b778-48e4-9e5b-56dbfb09d2e6">
